server/game/cards/13.3-PoS/EddardStark.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+
+class EddardStark extends DrawCard {
+	setupCardAbilities(ability) {
```

Comment:
The indentation for this file doesn't match up with the way we enforce code style. It would probably be productive to install an editor on your machine (we tend to use Visual Studio Code but others should work fine too), pull the code locally, and make sure your editor is set up to check the code style using `eslint`.

Reply:
I did this in Notepad++ but I guess a proper editor for Java Script would be a good idea to get.  
You do Spaces instead of Tabs?